### PR TITLE
Review app_settings

### PR DESCRIPTION
**Pull request by rubengrill**

### CODE REVIEW

File: requirements/common.txt [hunk 1]
```diff
@@ -1,31 +1,31 @@
-beautifulsoup4==4.4.0
-django-admin-sortable2==0.6.4
+beautifulsoup4==4.4.1
+django-admin-sortable2==0.6.7
 django-allauth==0.23.0
 django-angular==0.8.4
 django-appconf==1.0.1
 django-classy-tags==0.7.2
-django-cms==3.3.3
+https://github.com/jrief/django-cms/archive/f9b62031f64aa7d775f9d1befa7b6a70d0d1a719.zip#egg=django-cms
```
Comment (rfleschenberg) [inline requirements/common.txt]: Do we still need this? I'd prefer not to have the master branch depend on a private fork.

File: requirements/common.txt [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+django-mptt==0.8.7
 django-parler==1.6.4
 django-polymorphic==1.0.2
 django-post-office==2.0.8
 django-rest-auth==0.5.0
 django-sass-processor==0.5.3
 django-sekizai==0.10.0
 Django-Select2==5.8.9
-djangoshop-stripe==0.2.0
+https://github.com/awesto/djangoshop-stripe/archive/257a33f17df7f6d42e6d9815969ec6c801dcc66c.zip#egg=djangoshop-stripe
```
Comment (rfleschenberg) [inline requirements/common.txt]: Same question as with django-cms. If we need this state of djangoshop-stripe, we should release it first,

File: requirements/common.txt [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
 django-treebeard==4.1.0
 djangocms-bootstrap3==0.1.1
-djangocms-cascade==0.11.1
-djangocms-text-ckeditor==3.0.1
-djangorestframework==3.2.5
+https://github.com/jrief/djangocms-cascade/archive/70a03ccdc4b240cf928d497c2a76ce747c7fa560.zip#egg=djangocms-cascade
```
Comment (rfleschenberg) [inline requirements/common.txt]: Dito :)

File: requirements/common.txt (hunk 1 above)
Comment (jrief) [inline requirements/common.txt]: That's the problem I reported two weeks ag: https://github.com/divio/django-cms/pull/5809
Fortunately now it seems, as if the rejection of this patch is withdrawn, status was changed into "needs design decision".

Therefore this pull request shall be considered as step to the next release, presumably 0.10.
Nevertheless, it should become the master branch on which we can work upon.

File: requirements/common.txt (hunk 2 above)
Comment (jrief) [inline requirements/common.txt]: sure, but then I would publish a release which does not work with the current version of django-SHOP.

File: requirements/common.txt (hunk 3 above)
Comment (jrief) [inline requirements/common.txt]: Dito

**Comment by jrief** [comment]
So please don't focus on the dependency stuff, but rather on the refactoring.

File: shop/views/catalog.py [hunk 4]
```diff
@@ -196,7 +200,7 @@ class ProductSelectView(generics.ListAPIView):
     the data for rendering the select widget when looking up for a product.
     """
     renderer_classes = (JSONRenderer, BrowsableAPIRenderer)
-    serializer_class = ProductSelectSerializer
+    serializer_class = app_settings.PRODUCT_SELECT_SERIALIZER
```
Comment (rubengrill) [inline shop/views/catalog.py]: Does not exist (yet?)

**Pull request closed without merging**